lib/export-scala-regmap.js

```diff
@@ -450,7 +454,9 @@ const generateOMRegisterMap = (addressUnitBits, addressBlock) => {
   const registerMapElements = addressBlock.registers.map((register) => {
     const regFieldSeqElements = (register.fields || []).map(field => {
       const access = field.access || register.access || addressBlock.access;
-      return `${field.bitOffset} -> ` + generateRegisterField(field, 'Bool()', access);
+      const resetValue = isDefinedNumber(field.resetValue)
+        ? field.resetValue : register.resetValue;
```

Comment:
I don't think `resetValue` is a valid field of a DUH register, only fields can have it right?

Reply:
Ah, you're right. Sorry, I was getting this mixed up with IP-XACT, which does have it defined on the register in addition to the field. I'll fix this up.

Reply:
Just fixed this and force-pushed back up to this branch.